djstripe/stripe_objects.py

```diff
@@ -1097,6 +1190,76 @@ def _stripe_object_to_charge(cls, target_cls, data):
         if "charge" in data and data["charge"]:
             return target_cls._get_or_create_from_stripe_object(data, "charge")[0]
 
+    @classmethod
+    def upcoming(cls, api_key=settings.STRIPE_SECRET_KEY, customer=None, coupon=None, subscription=None, subscription_plan=None, subscription_prorate=None, subscription_proration_date=None, subscription_quantity=None, subscription_trial_end=None, **kwargs):
```

Comment:
use a double indent on this second row. If you want, three rows is okay here due to the sheer number of kwargs.